eng/native/configuretools.cmake

```diff
@@ -55,7 +55,7 @@ if(NOT WIN32 AND NOT CLR_CMAKE_TARGET_BROWSER)
     elseif(CMAKE_CROSSCOMPILING AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD AND (CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR
        CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL arm))
       set(TOOLSET_PREFIX "${TOOLCHAIN}-")
-    else()
+    elseif(CLR_CMAKE_TARGET_ALPINE_LINUX AND CMAKE_C_COMPILER_VERSION VERSION_LESS 8.0)
```

Comment:
`llvm-dev` package on Alpine has `llvm-objcopy` in later versions (checked with 8 and 10), but not in v5, which we are currently using the the CI.